### ISSUE TITLE
Rename mapping folders "to_mods" and "from_mods"

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/dor-services-app#3720]
Since mapping to and from MODS is going to be with us for a while, but Fedora is going away, we should change the name of these two folders in app/services/cocina

**Comment by justinlittman** [comment]
Agree, but I would like to do this as part of the post-migration code-cleanup.
